examples/00-core_examples/00-basic_dpf_implementation.py

```diff
@@ -0,0 +1,217 @@
+# operator only when the final operator is evaluated and the data is requested. So the order that you connect the
+# operators is important. There are three ways for doing this:
+
+#      4.1) By using the 'connect()' method derived from the 'Inputs' class, that the outputs pin of one operator to the inputs
+#      pin of another operator
+
+#       We will use the operator instantiated above "my_operator" and a new "my_other_operator"
+        my_other_operator.inputs.connect(my_operator.outputs) # Here the output of "my_operator" is connected
+#                                                               to the "my_other_operator" input
+#       or
+        my_other_operator.inputs.inputPinName.connect(my_operator.outputs.outputPinName)
+#  While this last approach is more verbose, it can be useful for operators having several matching inputs or outputs.
+
+#       An example with the "my_displacement_operator":
+#       We will use the case where we want to find the maximum value of the displacement field. So we instantiate an
+#       operator named "my_min_max_operator"
+        my_min_max_operator.inputs.connect(my_displacement_operator.outputs)
+#       or
+        my_min_max_operator.inputs.field.connect(my_displacement_operator.outputs.fields_container)
+
+#      4.2) By using the 'connect()' method from the own operator class. The difference here is that the input
+#      pin number needs to be specified
+
+#       We will use the operator instantiated above "my_operator" and a new "my_other_operator"
+        my_other_ooperator.connect(pin=inputPinNumber,inpt=my_operator,pin_out=outputPinNumber)
+#       Here the output of the "my_operator" is being connected to the "my_other_operator" input. So the inputPinNumber
+#       is the number o the "my_other_operator" pin where you tuo connect the output pin identified with "outputPinNumber"
+#       from "my_operator"
+
+#       An example with the "my_displacement_operator":
+#       We will use the case where we want to find the maximum value of the displacement field. So we instantiate an
+#       operator named "my_min_max_operator"
+        my_min_max_operator.connect(pin=0, input=my_displacement_operator, pin_out=0)
+
+#      4.3) By directly instantiating the new operator with the other operator as a parameter:
+
+#       We will use the 1.1 method
+        from ansys.dpf.core import operators as ops
+        my_other_operator = ops.subpackageAttribute.operatorAttribute(parameterName=my_operator)
+
+#       An example with the "my_displacement_operator":
+#       We will use the case where we want to find the maximum value of the displacement field.
+        from ansys.dpf.core import operators as ops
+        my_min_max_operator = ops.min_max.min_max(my_displacement_operator)
+
+######################################################################################################################
+# # ~~~~~~~ THE OPERATORS ~~~~~~~ # #
```

Comment:
A global comment: please use the RST ways of defining sections and titles. See https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst